### PR TITLE
fix typos in java documentation

### PR DESCRIPTION
**Pull request by garanews**
fix typos in java documentation

### CODE REVIEW

File: content/java/concepts/classes/classes.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Alice is 27.
 
 ## Abstract Classes
 
-Classes can also use the `abstract` keyword to supply common method implentations to multiple subclasses. Any class that contains abstraction (e.g. methods, fields, etc.), then it must also be abstract:
+Classes can also use the `abstract` keyword to supply common method implementations to multiple subclasses. Any class that contains abstraction (e.g. methods, fields, etc.), then it must also be abstract:
```
Comment (Dusch4593) [inline content/java/concepts/classes/classes.md:77]:
```suggestion
Classes can also use the `abstract` keyword to supply common method implementations to multiple subclasses. Any class that contains abstraction (methods, fields, etc.), then it must also be abstract:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@garanews Here are my suggestions for this PR:

Under `classes.md`:
- On line # 18, let's **bold** the first use of "classes" like so: `In Java, **classes** are ...`
- Let's add a comment above between lines # 47 and # 48 that says `// Person.java`.
- Let's fix the typo on line # 68 and change "contstructor" to "constructor".

Under `conditionals.md`:
- On line # 3, let's shorten the `Description` to (ideally) 150 characters or less.
- On line # 15, let's **bold** the word "Conditionals" (i.e., `**Conditionals** take ...`).

Under `constructors.md`:
- On line # 14, let's **bold** the word "Constructors" (i.e., `**Constructors** are like ...`).

Under `switch.md`:
- On line # 15, let's **bold** the word \`switch\` (i.e., `A **`switch`** statement ...`).
- On line # 23, under the "Syntax", let's change the code block signature from \```java to \```pseudo.
- Remove the content in lines # 36 through 57 since there is already an "Example" section with a \```java snippet near the bottom.
- On line # 66, let's add a `>` in front of `**Note:**`.

**Comment by garanews** [comment]
@Dusch4593 with your suggestion I've reduced the description of conditionals.md hope is fine :)
To bold the things, from a to **a**  is ok, but when there is `switch` should became **`switch`** or just **switch** ?  :P

**Comment by Dusch4593** [comment]
@garanews 

> @Dusch4593 with your suggestion I've reduced the description of conditionals.md hope is fine :)

Looks great, thank you for doing that. :)

> To bold the things, from a to a is ok, but when there is switch should became switch or just switch ? :P

Yup, we should leave the backticks in the bolded concept/term. 👍🏻 

> Another question about Note: to be modified into >Note:
> If it is a crossed action, flat on all documentation should I apply this like 170 times present in 150 files: I can do it, just tell me :)

If I understand this correctly, you're asking about applying blockquotes to all the **Note:**s throughout all Docs content? Let me check with my team on this one just because of how files would be involved. 😄

File: content/java/concepts/switch/switch.md [hunk 2]
```diff
@@ -63,7 +42,7 @@ switch (grade) {
 
 In the code above, suppose grade is equal to `10`, then the output would be “Sophomore”.
 
-**Note:** Without the `break` keyword at the end of each case, the program would execute the code for the first matching case and _all_ subsequent cases, including the `default` code. This behavior is different from `if`/`else` conditional statements which execute only one block of code.
+>**Note:** Without the `break` keyword at the end of each case, the program would execute the code for the first matching case and _all_ subsequent cases, including the `default` code. This behavior is different from `if`/`else` conditional statements which execute only one block of code.
```
Comment (Dusch4593) [inline content/java/concepts/switch/switch.md:45]:
```suggestion
> **Note:** Without the `break` keyword at the end of each case, the program would execute the code for the first matching case and _all_ subsequent cases, including the `default` code. This behavior is different from `if`/`else` conditional statements which execute only one block of code.
```

File: content/java/concepts/classes/classes.md [hunk 3]
```diff
@@ -74,7 +75,7 @@ Alice is 27.
 
 ## Abstract Classes
 
-Classes can also use the `abstract` keyword to supply common method implentations to multiple subclasses. Any class that contains abstraction (e.g. methods, fields, etc.), then it must also be abstract:
+Classes can also use the `abstract` keyword to supply common method implementations to multiple subclasses. Any class that contains abstraction (methods, fields, etc.), then it must also be abstract:
 
 ```java
 abstract class Person {
```
Comment (Dusch4593) [inline content/java/concepts/classes/classes.md:81]:
```suggestion
// Person.java
abstract class Person {
```

File: content/java/concepts/classes/classes.md [hunk 4]
```diff
@@ -45,6 +45,7 @@ Every instance has access to its own set of variables known as instance fields.
 For example, `Bob` and `Alice` may each be defined as instances of the class called `Person` with the `new` keyword:
 
 ```java
+//Person.java
```
Comment (Dusch4593) [inline content/java/concepts/classes/classes.md:48]:
```suggestion
// Person.java
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@garanews Just three final, super small suggestions for my review. 😄

File: content/java/concepts/switch/switch.md (hunk 2 above)
Comment (garanews) [reply content/java/concepts/switch/switch.md:45]: yes exactly, this one if it is the "new writing style" (adding > before Note ) should be applied on many files!

**Review by Dusch4593 — APPROVED** [review]
@garanews This LGTM! 👍🏻

What say you, @SSwiniarski?

**Comment by Dusch4593** [comment]
@SSwiniarski Does this one look good for merge?

**Review by SSwiniarski — APPROVED** [review]
@garanews @Dusch4593 Looks good.

**Pull request merged**